Pax.cs

```diff
@@ -109,7 +109,8 @@ public class Frontend {
       Console.ResetColor();
 
       // FIXME accept a -j parameter to limit number of threads?
-      Parallel.ForEach(PaxConfig.deviceMap, device => device.Capture());
+      foreach (var device in PaxConfig.deviceMap)
+        device.StartCapture();
```

Comment:
I suspected there might be a problem with threading, and I think I saw `StartCapture()` used in some example code. It turned out to be non-blocking, so I gave it a try. The need to set `StopCaptureTimeout` was just a guess when the program didn't want to finish after shutdown.